doc/book/admin/upgrades/1.6-1.10.rst

```diff
@@ -0,0 +1,59 @@
+..  _admin-upgrades-1.6-1.10:
+
+Live upgrade from Tarantool 1.6 to 1.10
+=======================================
+
+This page includes explanations and solutions to some common issues
+when upgrading a replica set from Tarantool 1.6 to 1.10.
+
+..  include:: ../_includes/1.6-to-2.x-condition.rst
+
+Let's first reiterate the upgrade procedure for any Tarantool version:
+
+..  include:: ../_includes/upgrade_procedure.rst
+
+7.  Run ``box.schema.upgrade()`` on the new master
+    as described in the section ":ref:`Finishing the upgrade <admin-upgrades_db>`".
+    There is no need  to run ``box.schema.upgrade()`` on every node:
+    changes are propagated to other nodes via the regular replication mechanism.
+
+8.  Run ``box.snapshot()`` on every node in the replica set
+    to make sure that the replicas immediately see the upgraded database state in case of restart.
+
+What's different when upgrading from Tarantool 1.6:
+
+**Step 2:** Tarantool 1.10+ fails to recover from 1.6 xlogs, unless ``box.cfg{force_recovery = true}`` is set.
+There is some small difference between 1.6 and 1.10 xlogs, which makes 1.6 xlogs appear erroneous to 1.10+ instances.
+In order to work around this, start the instance in ``force_recovery`` mode.
```

Comment:
> In order to work around this, start the instance in ``force_recovery`` mode.

When and where a user should set ``box.cfg{force_recovery = true}`` ? In the init.lua starting file before launching the instance after upgrade? Better to make it more clear.